convert.py

```diff
@@ -1037,6 +1037,10 @@ def load_some_model(path: Path) -> ModelPlus:
     if path.is_dir():
         globs = ["consolidated.00.pth", "pytorch_model-00001-of-*.bin", "*.pt"]
         files = [file for glob in globs for file in path.glob(glob)]
+        if not files:
+            # Check if it's a set of safetensors files
+            globs = ["model-00001-of-*.safetensors"]
```

Comment:
Why not just add `"model-00001-of-*.safetensors"` to the `globs` above?

Reply:
That would generally work as well. I was thinking to treat it the same as the ggml files, where it's lower priority than the pt files, in case both exist. If I simply add the safetensors pattern to the globs, and both are present, the convert script will abort, since it's unsure which one to use.
In any case, I'm happy to change it if you prefer.

Reply:
That sounds good to me.
I initially wanted to minimize the behavior change, but I've updated the PR to try with the safetensors first.